.github/utils/pydoc-markdown.sh

```diff
@@ -13,7 +13,7 @@ done
 cd ..
 rm -rf temp-preview && mkdir temp-preview
 cd temp-preview
-for file in ../config-preview/* ; do
+for file in ../config/* ; do
```

Comment:
We're doing the same here twice now. Since this script previously worked for preview markdown docs and 1.x docs.
Let's remove everything after line 11.